packages/next-templates/src/components/TopTask/TopTaskHeader.css

```diff
@@ -0,0 +1,7 @@
+.example-toptask-header {
```

Comment:
Zou een `toptask-header` altijd 1/3 breed zijn, of is dat eigenlijk afhankelijk van de context en zou dat flexibel moeten zijn?